packages/office-addin-lint/src/lint.ts

```diff
@@ -20,19 +21,20 @@ function normalizeFilePath(filePath: string): string {
   return filePath.replace(/ /g, "\\ "); // Converting space to '\\'
 }
 
-function getEsLintBaseCommand(): string {
-  const eslintBaseCommand: string = `node ${esLintFilePath} -c ${esLintConfigPath} --resolve-plugins-relative-to ${__dirname}`;
+function getEsLintBaseCommand(useTestConfig: boolean): string {
```

Comment:
Can we put a default value for `useTestConfig` here to false?
It would be great to not need to always specify that it is not a test.